projects/1.18/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -936,39 +982,127 @@
-  "ars_nouveau.apparatus.norecipe": "找不到配方。基座必须放置在3格内。",
-  "ars_nouveau.imbuement.norecipe": "找找不到配方。合成需要的基座必须放置在灌注室1格范围内。",
+  "ars_nouveau.page.dowsing_rod": "使用后能短暂地占卜紫水晶的位置，并使得使用者75格范围内的魔法生物发光。单个占卜杖的使用次数有限。",
+  "ars_nouveau.apparatus.norecipe": "找不到配方。基座必须被放置在3格内。",
+  "ars_nouveau.imbuement.norecipe": "找不到配方。合成需要的基座必须放置在灌注室1格范围内。",
   "ars_nouveau.spell_book_gui.select": "选择",
   "ars_nouveau.levels_required": "需要等级：%s",
   "ars_nouveau.crafting": "合成：%s",
   "ars_nouveau.all_glyphs": "全部魔符",
   "ars_nouveau.not_enough_exp": "合成魔符所需经验不足。",
   "ars_nouveau.exp": "需要经验等级：%s",
   "ars_nouveau.scribing": "抄写法术",
-  "ars_nouveau.home_set": "已设定家园。",
+  "ars_nouveau.home_set": "已设置家园。",
   "ars_nouveau.gathering_at": "聚集于：%s",
   "entity.ars_nouveau.amethyst_golem": "紫晶傀儡",
   "tooltip.ars_nouveau.amethyst_charm": "通过在紫水晶母岩旁施展启蒙仪式获得",
   "item.ars_nouveau.amethyst_golem_charm": "紫晶傀儡护符",
   "ars_nouveau.page1.amethyst_golem_charm": "紫晶傀儡会种植并收获家园附近的紫水晶。在紫晶母岩旁施展启蒙仪式可以制造紫晶傀儡，其死亡后会掉落紫晶护符。",
   "ars_nouveau.page2.amethyst_golem_charm": "",
-  "ars_nouveau.page3.amethyst_golem_charm": "对方块使用紫晶护符可以召唤紫晶傀儡。在让紫晶傀儡进行紫晶种植前，必须先设定它们的家园。手持支配之杖，先右击傀儡再右击方块，紫晶傀儡就会将方块10格内视为家园。",
+  "ars_nouveau.page3.amethyst_golem_charm": "对方块使用紫晶护符可以召唤紫晶傀儡。手持支配之杖，先右击傀儡再右击方块，紫晶傀儡就会将此地视为家园，并在家园的10格范围内工作。只有设置了家园的紫晶傀儡才会正常工作。",
   "ars_nouveau.amethyst_farming": "紫晶种植",
-  "ars_nouveau.page4.amethyst_golem_charm": "设定家园后，紫晶傀儡会开始进行紫晶种植。它们会将紫水晶块转换成紫水晶母岩、收获紫水晶簇、加快母岩产生晶芽的速度、拾取并存储紫水晶碎片。",
+  "ars_nouveau.page4.amethyst_golem_charm": "设置家园后，紫晶傀儡会开始种植紫晶。它们能将紫水晶块变成母岩，加速紫水晶芽的生长，挖掘成熟的紫水晶簇，并收集掉落的紫水晶碎片。",
   "ars_nouveau.amethyst_storage": "存储紫水晶",
-  "ars_nouveau.page5.amethyst_golem_charm": "如果紫晶傀儡的家园被设定在容器上，它们会将捡起的紫水晶碎片存放进去。而如果家园没有设定在容器上，它们则只会忽略地上的物品。",
+  "ars_nouveau.page5.amethyst_golem_charm": "如果紫晶傀儡的家园被设定在容器上，它们会将捡起的紫水晶碎片存放进去。否则它们就不会收集掉落的紫水晶碎片。",
   "ars_nouveau.recorded_codex": "你在手抄本里记下了你知道的魔符。",
-  "ars_nouveau.updated_codex": "你在手抄本里更新了%s个魔符。",
+  "ars_nouveau.updated_codex": "你向手抄本更新了%s个魔符。",
   "ars_nouveau.consumed_codex": "你从手抄本里学习了%s个魔符。",
   "ars_nouveau.codex_no_use": "从这本手抄本里你学不到任何新知识。",
-  "ars_nouveau.codex_up_to_date": "手抄本已经是最新的。",
+  "ars_nouveau.codex_up_to_date": "手抄本内容已更新至最新",
   "ars_nouveau.codex_not_enough_exp": "记录魔符所需的经验不足。至少需要%s级。",
   "ars_nouveau.recorded_by": "由%s撰写",
   "ars_nouveau.contains_glyphs": "含有%s个魔符。",
-  "ars_nouveau.codex_tooltip": "用于记录你的魔符。",
+  "ars_nouveau.codex_tooltip": "用于记录并分享分享你的魔符。",
   "item.ars_nouveau.annotated_codex": "注解手抄本",
-  "ars_nouveau.page.annotated_codex": "注解手抄本能用来向其他人分享你知道的魔符。右击使用手抄本就能消耗经验写下魔符。知道的魔符越多，写下魔符需要的经验也越多。再次使用手抄本能更新其中记录的已知魔符。其他玩家能使用手抄本来学习其中的魔符，不过学习后手抄本会被消耗。"
+  "ars_nouveau.page.annotated_codex": "注解手抄本能用来向其他人分享你知道的魔符。右击使用手抄本就能记录已知的全部魔符，并根据本次记录的魔符数量消耗经验。再次使用手抄本可以更新其中内容。其他玩家能消耗手抄本来学习其中的魔符。",
+  "ars_nouveau.ritual.no_start": "启动当前仪式或破坏此方块来终止仪式",
+  "ars_nouveau.lights_on": "动态光源已开启。这将在CPU和内存配置较低的电脑上引起卡顿，并有可能在与类似“optifine”的“优化模组”同时使用时产生未知错误。执行这个指令或者修改ars_nouveau-client配置文件来关闭动态光源。",

```

Optifine，专有名词首字母大写；并且这里的引号没有什么作用
L1039同理
```suggestion
  "ars_nouveau.lights_on": "动态光源已开启。这将在CPU和内存配置较低的电脑上引起卡顿，并有可能在与类似Optifine的优化模组同时使用时产生未知错误。执行这个指令或者修改ars_nouveau-client配置文件来关闭动态光源。",
```